src/selectors/nodes.js

```diff
@@ -16,12 +16,16 @@ const getNodeTags = state => state.node.tags;
 const getNodeType = state => state.node.type;
 const getNodeLayer = state => state.node.layer;
 const getHoveredNode = state => state.node.hovered;
-const getClickedNode = state => state.node.clicked;
 const getTagActive = state => state.tag.active;
 const getTextLabels = state => state.textLabels;
 const getFontLoaded = state => state.fontLoaded;
 const getNodeTypeDisabled = state => state.nodeType.disabled;
 
+/**
+ * Gets the currently clicked node if any
+ */
+export const getClickedNode = state => state.node.clicked;
```

Comment:
This doesn't need to be exported from this file and imported in the other, as it's not a true complex selector, it's just a simple prop selector function like the ones on lines 12-22. I'd put it with those and replicate it in the files where it's imported. The reason I say this is that it's annoyingly easy to introduce circular dependencies in the selector files and I'd prefer not to risk this more than necessary. I know this file's functions are already being imported in `src/selectors/metadata.js` but I'd still prefer to avoid unnecessary imports tbh. Does this make sense?

Reply:
I think ideally even these should be shared from a maintainability point of view, but if this causes circular dependencies then sure I'll make a copy instead. Going forwards, it might be possible to refactor all of these into a separate shared file which might avoid those too, but for now I've done this.

Reply:
I hear you, I think I could go either way on them. Tbh though I see that as so simple that it doesn't seem hugely worthwhile to abstract them? But yeah if we were going to, then putting them in a single share file would make the most sense. But that should be a separate PR. Thanks!